ivy/functional/backends/torch/linear_algebra.py

```diff
@@ -521,3 +521,33 @@ def vector_to_skew_symmetric_matrix(
 
 
 vector_to_skew_symmetric_matrix.support_native_out = True
+
+
+def multi_dot(
```

Comment:
This function should not be here, if you want to add this you have to add it in the experimental folder for the functional API.
Moreover, adding a function in the backend is a separate task actually which requires adding it's test in the testing respective files as well.